sdk/nodejs/yaml/yaml.ts

```diff
@@ -188,7 +184,7 @@ import * as outputs from "../types/output";
 
         if (config.objs !== undefined) {
             const objs= Array.isArray(config.objs) ? config.objs: [config.objs];
-                const docResources = parseYamlDocument({objs: objs, transformations: config.transformations}, opts);
+                const docResources = parseYamlDocument({objs: pulumi.output(objs), transformations: config.transformations}, opts);
```

Comment:
Per other comments, you probable want to do this conversion to an `Output` inside `parseYamlDocment` and let callers pass in whatever they wants (i.e. `Input`).  That is the standard Pulumi API design approach (inputs are `Input`, outputs are `Output`).